device-types/Huawei/CE6856-48T6Q-HI.yaml

```diff
@@ -0,0 +1,135 @@
+---
+manufacturer: Huawei
+model: CE6856-48T6Q-HI
+slug: huawei-ce6856-48t6q-hi
+part_number: 02351LVC
+u_height: 1
+is_full_depth: false
+comments: '[Huawei CE6856-48T6Q-HI] (https://support.huawei.com/enterprise/en/doc/EDOC1000019246/5e0ec9b2/ce6856-48t6q-hi)'
+weight: 12.6
+weight_unit: kg
+console-ports:
+  - name: console0
+    type: rj-45
+module-bays:
+  - name: FAN1
+  - name: FAN2
+power-ports:
+  - name: PWR1
+    type: iec-60320-c14
+    maximum_draw: 600
+  - name: PWR2
+    type: iec-60320-c14
+    maximum_draw: 600
```

Comment:
Hello, thanks for the contribution. It appears that these PSU's are modular. These should be defined under `module-bays` rather than `power-ports`, and then the associated PSU modules should be created within the module-types folder if they do not already exist. Thank you.